README.md

```diff
@@ -1,2 +1,97 @@
-# CC-Global-Summit
-Website made in base of Cindy Shin's design posted in behance
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+# 📖 Yearly World Comedian Conference <a name="about-project"></a>
+
+**Yearly World Comedian Conference** is a website where you will be able to see all the information regarding to the Yearly World Comedian Conference, from the date and place it's gonna be done this year to our logo's origin.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">Javascript</a></li>
+    <li><a href="#">HTML</a></li>
+    <li><a href="#">CSS</a></li>
+    <li><a href="#">VS Code</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">Github pages</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **You can use the navbar to move through the different pages of the site**
+- **It is responsive for desktop and mobile screen sizes**
+- **Some parts are dinamically generated with Javascript**
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://www.loom.com/share/4685dac778c7469fb5d9b3a97d199be6)
```

Comment:
The live demo link must be the link to your deployed web page, not to your presentation video. Kindly make sure to update this.